assets/js/request-demo.js

```diff
@@ -105,15 +105,16 @@ $(function() {
       elementSC1.setAttribute("type", "hidden");
       form.appendChild(elementSC1);
 
-      snowplow(function () {
-
-        var elementDUID = document.createElement("input");
-        elementDUID.name = "00N2400000HRtrl";
-        elementDUID.value = this.snplow5.getDomainUserId();
-        elementDUID.setAttribute("type", "hidden");
-        form.appendChild(elementDUID);
-
-      })
+      try {
+        snowplow(function () {
+          var elementDUID = document.createElement("input");
+          elementDUID.name = "00N2400000HRtrl";
+          elementDUID.value = this.snplow5.getDomainUserId();
+          elementDUID.setAttribute("type", "hidden");
+          form.appendChild(elementDUID);
+        })
+      }
+      catch (e) {}
```

Comment:
@yalisassoon here we could report this catch somewhere. 
If you have one place in mind, I could add it otherwise, we're fine and we just move forward.